### PR TITLE
Adding request and response types

### PR DESCRIPTION
**Pull request by vitarb**
This PR adds some basic request/reply types that will be used for version checking.

### CODE REVIEW

File: check/response.go [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+type ReleaseInfo struct {
+	Version     string `json:"version"`
+	ReleaseDate int64  `json:"release_date"`
+	Notes       string `json:"notes"`
+}
+
+type VersionCheckResponse struct {
+	Current     ReleaseInfo `json:"current"`
+	Recommended ReleaseInfo `json:"recommended"`
+	Severity    Severity    `json:"important"`
```
Comment (alexshtin) [inline check/response.go]:
```suggestion
	Severity    Severity    `json:"severity"`
```
?

File: check/request.go [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package check
+
+type VersionCheckRequest struct {
+	Product string
+	Version string
+	Arch    string
+	OS      string
+	DB      string
```
Comment (alexshtin) [inline check/request.go]: Run `golint` on it. It might have different opinion about case here.

File: check/request.go [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+	Token   string
```
Comment (alexshtin) [inline check/request.go]:
```suggestion
	Token   []byte
```
?

**Review by alexshtin — APPROVED** [review]

File: check/response.go (hunk 1 above)
Comment (vitarb) [reply check/response.go]: Yes, changed names a few times already.

File: check/request.go (hunk 3 above)
Comment (vitarb) [reply check/request.go]: It gets passed as a get URL parameter, I would prefer to keep it a string.

File: check/request.go (hunk 2 above)
Comment (vitarb) [reply check/request.go]: `golint` complains only about missing comments, which I'm going to fix.

File: check/request.go (hunk 3 above)
Comment (alexshtin) [reply check/request.go]: ok

**Review by alexshtin — APPROVED** [review]

**Pull request merged**